assets/js/theme/common/models/forms.js

```diff
@@ -1,6 +1,6 @@
 const forms = {
     email(value) {
-        const re = /^.+@.+\..+/;
+        const re = /^\S+@\S+\.\S+/;
```

Comment:
Why do we use capital S here?

Reply:
it is [used](https://www.w3schools.com/jsref/jsref_regexp_whitespace_non.asp) to exclude a non-whitespace characters in email